src/DeferredFutures.jl

```diff
@@ -47,6 +47,21 @@ function Base.show(io::IO, ref::DeferredFuture)
     print(io, "$(typeof(ref).name.name) at ($(rc.where),$(rc.whence),$(rc.id))")
 end
 
+"""
+    serialize(s::AbstractSerializer, ref::DeferredFuture)
+
+Serialize a DeferredFuture such that it can de deserialized by `deserialize` in a cluster.
+"""
+function Base.serialize(s::AbstractSerializer, ref::DeferredFuture)
+    if VERSION < v"0.6.0-rc1"
```

Comment:
it's not wrong, but that's a commit from a branch - there are a bunch of higher-versioned master commits from before the PR was merged that won't include the change